tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -222,6 +224,108 @@ def test_non_org_rename_403(self):
         self.get_error_response(other_org.slug, "old_slug", status_code=403)
 
 
+class ProjectUpdateTestTokenAuthenticated(APITestCase):
+    endpoint = "sentry-api-0-project-details"
+    method = "put"
+
+    def setUp(self):
+        super().setUp()
+        self.project = self.create_project(platform="javascript")
+        self.user = self.create_user("bar@example.com")
+
+    def test_member_updates_denied_with_token(self):
+        self.create_member(
+            user=self.user,
+            organization=self.project.organization,
+            teams=[self.project.teams.first()],
+            role="member",
+        )
```

Comment:
@mdtro For completeness, we probably want to confirm if `project:read` scope still works as usual (the happy path).